src/postgres/tree/PostgreSQLAccountTreeItem.ts

```diff
@@ -0,0 +1,79 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { Database, Server } from 'azure-arm-postgresql/lib/models';
+import * as vscode from 'vscode';
+import { AzureParentTreeItem, AzureTreeItem, parseError } from 'vscode-azureextensionui';
+import { getThemeAgnosticIconPath, Links } from '../../constants';
+import { ClientConfigClass } from '../ClientConfigClass';
+import { IPostgreSQLTreeRoot } from './IPostgreSQLTreeRoot';
+import { PostgreSQLDatabaseTreeItem } from './PostgreSQLDatabaseTreeItem';
+
+export class PostgreSQLAccountTreeItem extends AzureParentTreeItem<IPostgreSQLTreeRoot> {
+    public static contextValue: string = "cosmosDBPostgresServer";
+    public readonly contextValue: string = PostgreSQLAccountTreeItem.contextValue;
+    public readonly childTypeLabel: string = "Database";
+    public readonly id: string;
+    public readonly label: string;
+    public readonly connectionString: string;
+    public readonly clientConfig: ClientConfigClass;
+    public readonly host: string;
+
+    private _root: IPostgreSQLTreeRoot;
+
+    constructor(parent: AzureParentTreeItem, id: string, label: string, isEmulator: boolean, readonly databaseAccount?: Server, readonly databases?: Database[], connectionString?: string) {
+        super(parent);
+        this.id = id;
+        this.label = label;
+        if (isEmulator) {
+            this.connectionString = connectionString;
+        }
+        this._root = Object.assign({}, parent.root, { isEmulator });
+        this.host = databaseAccount.fullyQualifiedDomainName;
+    }
+
+    // overrides ISubscriptionContext with an object that also has Mongo info
+    public get root(): IPostgreSQLTreeRoot {
+        return this._root;
+    }
+
+    public get iconPath(): string | vscode.Uri | { light: string | vscode.Uri; dark: string | vscode.Uri } {
+        return getThemeAgnosticIconPath('CosmosDBAccount.svg');
```

Comment:
You should get a Postgres icon for this instead of cosmos. See here: https://cds-icons.azurewebsites.net/

Reply:
So Postgres structure is Server > Database > Schema > Table...Is it necessary to change the icons for Schema and Table as well?

Reply:
That would be great if you can find them. But yeah my only _requirement_ is changing the icon here in `PostgreSQLAccountTreeItem.ts`